packages/android_intent/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: android_intent
 description: Flutter plugin for launching Android Intents. Not supported on iOS.
 homepage: https://github.com/flutter/plugins/tree/master/packages/android_intent
-version: 0.3.6
+version: 0.3.7
```

Comment:
nit: I think this should actually be a patch bump, semver considers transitive dependency upgrades a small change since they don't upgrade the public API.

https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api

Reply:
OK